examples/quickstart/jupyter-notebooks/notebooks/03-query/04-UnionOperations.ipynb

```diff
@@ -0,0 +1,644 @@
+    "  \"Flight_Number_Reporting_Airline\"\n",
+    "  FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "  WHERE Origin = 'ATL'\n",
+    "  AND DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-01 11:00:00'\n",
+    "  ORDER BY __time\n",
+    "  LIMIT 10\n",
+    "  )\n",
+    "  \n",
+    "SELECT * from set1\n",
+    "UNION ALL\n",
+    "SELECT * from set2\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5e77fa9-a60c-4955-b763-58d970d7326d",
+   "metadata": {},
+   "source": [
+    "This is what's known as a [top-level](https://druid.apache.org/docs/latest/querying/sql.html#top-level) `UNION` operation: each set of results was gathered individually, one after the other, and the list of results concatenated.\n",
+    "\n",
+    "Notice that these results are not in order by time – even though the individual sets did `ORDER BY` time. Druid has simply concatenated the two result sets together.\n",
+    "\n",
+    "The following cell contains another valid use of `UNION ALL`, where some filtering and `GROUP BY` operations have been added. Notice there is a mix of this being done in the sets themselves and as part of the top level `UNION`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3514ad80-cbe6-4563-8b7b-9d48f8b9a508",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "WITH\n",
+    "set1 AS (\n",
+    "    SELECT \"Reporting_Airline\", \"Distance\"\n",
+    "    FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "    WHERE DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-01 11:00:00'\n",
+    "  ),\n",
+    "set2 AS (\n",
+    "    SELECT\n",
+    "        \"Reporting_Airline\",\n",
+    "        COUNT(*) AS \"Frights\",\n",
+    "        MAX(Distance) AS \"Lengthiest\",\n",
+    "        MIN(Distance) AS \"Shortest\"\n",
+    "    FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "    WHERE DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-02 11:00:00'\n",
+    "    AND \"Reporting_Airline\" LIKE 'A%'\n",
+    "    GROUP BY 1\n",
+    "  )\n",
+    "\n",
+    "SELECT \"Reporting_Airline\",\n",
+    "        COUNT(*) AS \"Frights\",\n",
+    "        MAX(Distance) AS \"Lengthiest\",\n",
+    "        MIN(Distance) AS \"Shortest\" from set1\n",
+    "  WHERE \"Reporting_Airline\" LIKE 'AA'\n",
+    "  GROUP BY 1\n",
+    "UNION ALL\n",
+    "SELECT * from set2\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1beb4ad7-93cf-4922-9f57-d7b9739fea4a",
+   "metadata": {},
+   "source": [
+    "Optionally, run the next cell – it shows the precise `EXPLAIN PLAN` for the query. You can see there are two `query` exeuction plans, one for each of our queries, and that Druid's planning process has taken time to optimize how the query above will actually execute."
```

Comment:
Possibly worth having a brief description of what EXPLAIN PLAN means? Instead or in addition, you could mention it's how Druid converts the SQL query into a native one